### PR TITLE
[10793] Collection of data and notification to listeners SUBSCRIPTION_THROUGHPUT

### PR DESCRIPTION
**Pull request by MiguelBarro**
Merge after https://github.com/eProsima/Fast-DDS/pull/1955

### CODE REVIEW

File: include/fastdds/statistics/rtps/StatisticsCommon.hpp [hunk 1]
```diff
@@ -244,6 +247,13 @@ class StatisticsReaderImpl
      */
     void on_nackfrag(
             int32_t count);
+
+    /**
+     * @brief Reports throughtput based on last added sample to history
```
Comment (JLBuenoLopez-eProsima) [inline include/fastdds/statistics/rtps/StatisticsCommon.hpp:252]:
```suggestion
     * @brief Reports subscription throughtput based on last added sample to reader's history
```

File: include/fastdds/statistics/rtps/StatisticsCommonEmpty.hpp [hunk 2]
```diff
@@ -131,6 +140,15 @@ class StatisticsReaderImpl
     {
     }
 
+    /**
+     * @brief Reports throughtput based on last added sample to history
```
Comment (JLBuenoLopez-eProsima) [inline include/fastdds/statistics/rtps/StatisticsCommonEmpty.hpp:144]:
```suggestion
     * @brief Reports subscription throughtput based on last added sample to reader's history
```

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM! Only a suggestion to improve the documentation

**Review by JLBuenoLopez-eProsima — DISMISSED** [review]
:ok_hand:

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM!

**Comment by MiguelCompany** [comment]
Not waiting for test results, as the last ones were ok, and changes after them are just a rebase after merging #1955, which this PR already included.

We'll leave tests for #1956, which will be rebased after this one is merged.

**Pull request merged**